tslint.json

```diff
@@ -2,8 +2,12 @@
   "rulesDirectory": [
     "codelyzer"
   ],
+  "extends": [
```

Comment:
This will require the npm package to be installed, but I don't see updates to package.json or package-lock.json in these changes?